eslint.config.js

```diff
@@ -0,0 +1,8 @@
+/*
```

Comment:
why not `/*!` ?

Reply:
Yes its more like most tools I know specially handle `*!` as legal comment and e.g. preserve it during minification.